meta-ostro/conf/distro/ostro.conf

```diff
@@ -144,7 +144,7 @@ DISTRO_EXTRA_RRECOMMENDS += " ${OSTRO_DEFAULT_EXTRA_RRECOMMENDS}"
 QEMU_TARGETS ?= "arm i386 x86_64"
 
 # Set PREMIRRORS to prioritize Ostro OS autobuilder source cache.
-# The OSTRO_SOURCE_MIRROR_URL can be overriden in local.conf. 
+# The OSTRO_SOURCE_MIRROR_URL can be overriden in local.conf.
```

Comment:
Please rebase on top of current master. While you are doing that, can you take out this unrelated whitespace change?